cmd/fleet/serve.go

```diff
@@ -636,10 +622,13 @@ the way that the Fleet server works.
 				initFatal(err, "failed to register stats schedule")
 			}
 
-			if err := cronSchedules.StartCronSchedule(func() (fleet.CronSchedule, error) {
-				return newVulnerabilitiesSchedule(ctx, instanceID, ds, logger, &config.Vulnerabilities)
-			}); err != nil {
-				initFatal(err, "failed to register vulnerabilities schedule")
+			if !config.Vulnerabilities.ExternalScheduled {
+				// vuln processing by default is run by internal cron mechanism
+				if err := cronSchedules.StartCronSchedule(func() (fleet.CronSchedule, error) {
+					return newVulnerabilitiesSchedule(ctx, instanceID, ds, logger, &config.Vulnerabilities)
+				}); err != nil {
+					initFatal(err, "failed to register vulnerabilities schedule")
+				}
 			}
```

Comment:
Nit: Let's add a info log when vulnerabilities cron is off because it's configured to run externally? (Useful for troubleshooting invalid configs on multiple fleet instances, etc.)

Reply:
added a message